### DISCUSSION TITLE
Can a chart implement dragging of a specific data point?

### DISCUSSION THREAD

**Discussion by JusterZhu** [Live-Charts/LiveCharts2#1591] [Q&A]
I am using the CartesianChart control from LiveCharts2 in a WPF application. I would like to implement the ability to drag any data point with the mouse to modify its position.

**Comment by beto-rodriguez** [comment]
Hi, this is an interesting example; to achieve that, you first need to know which point was clicked:

https://livecharts.dev/docs/Maui/2.0.0-rc2/samples.events.cartesian

Then when the pointer goes up, you must convert the position to the chart coordinate system:

https://livecharts.dev/docs/Maui/2.0.0-rc2/samples.events.addPointOnClick

That should be the way, but let me know if there is something else, I can help you with, I will move this to a discussion in case you need further assistance.

**Reply by g0g8rz** [reply]
Hi @beto-rodriguez. I also need to implement point dragging, so this question interests me.  Your answer says "Then when the pointer goes up..." but how do we know when the pointer goes up? There does not seem to be a ChartPointPointerUp or DataPointerUp event implemented.  

Thanks!
